### ISSUE TITLE
Handle quotes in image alt text

### ISSUE THREAD

**Issue by daughtkom** [noinnion/greader#383]
XKCD.com web comic puts an extra punchline in alt text of the image. Sometimes this caption contains quotes, and when it does it cannot be viewed.

**Comment by noinnion** [comment]
Long click on the image, and select Show Caption

**Comment by daughtkom** [comment]
I stopped using this app a year ago because of this, so I cannot reproduce it any more, but it's very to see for yourself.  Subscribe to XKCD, then try to view the alt/title text of any day's comic in the reader app.  The author regularly uses quotes in the hover text, so it shouldn't take long to find one with the issue, including yesterday's: https://xkcd.com/2424/.

**Comment by noinnion** [comment]
Unfortunately it is not possible fix it because of technical limitation. But thanks for your feedback.

**Comment by daughtkom** [comment]
I'm sure an appropriate call to `string.replace(...)` would do the trick.  The quote character is escaped in the feed already:

![image](https://user-images.githubusercontent.com/1824392/108120434-1a8eda00-7056-11eb-8be5-79fbd1c9ec49.png)

It only needs to be unescaped before rendering.

Your app, your call, though.  As I mentioned, I've already had to move on to another reader because this is broken (other things).  

Good luck getting back into it after several years.  As a developer myself, I'm sure returning to an old project is not the most exciting thing on your plate.
